### PR TITLE
update to latest versions of dependencies

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

File: .bowerrc [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-    ]
-  }
-}
+	"registry": {
+		"search": [
+			"https://origami-bower-registry.ft.com",
+			"https://registry.bower.io"
+		]
+	}
+}
```
Comment (rowanmanning) [inline .bowerrc]: Not even sure why this is in the repo... Should be fine to remove? It doesn't use Bower

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
-  - "5"
```
Comment (rowanmanning) [inline .travis.yml]: Technically removing these = breaking change. Not adverse to that, just needs updating in the README too

File: .bowerrc (hunk 1 above)
Comment (JakeChampion) [reply .bowerrc]: It is used in the tests to install o-icons

Comment (rowanmanning) [reply .bowerrc]: :D my bad

**Pull request merged**